example_package/README.md

```diff
@@ -65,29 +66,39 @@ Jest to klasa używana w plikach `.tex`.
 Nadaje ona odpowiedni wygląd dokumentom.
 Automatycznie tworzy ona nagłówki i stopki.
 Wystarczy że stworzymy treść dokumentu pomiędzy znacznikami `\start` i `\finish`.
-Dodatkowo udostępnia następujące funkcje:
+Dodatkowo udostępnia następujące funkcje.
+Zmienne podane w `{}` są obowiązkowe a w `[]` są opcjonalne, można ich nie podawać wcale lub tylko jakiś prefiks.
+Przykładowo `\img{a}`, `\img[0.5]{a}` lub `\img[0.5][obrazek 1]{a}`.
 
 - `\tc{n}` - Stylizuje podany tekst na talentowy kolor.
-- `\plainimg{img1.jpg}` - Wstawia obrazek o podanej ścieżce.
-- `\img{img1.jpg}{opis}{t/b}` - Wstawia obrazek o podanej ścieżce z opisem u góry lub na dole.
-Można też `\timg{img1.jpg}{opis}`, `\bimg{img1.jpg}{opis}`.
-- `\start{}` - Rozpoczyna treść, musi być na samym początku treści dokumentu.
-- `\finish{}` - Kończy treść, musi być na samym końcu treści dokumentu.
+- `\start` - Rozpoczyna treść, musi być na samym początku treści dokumentu.
```

Comment:
Doprecyzowałbym że po nagłówach, bo "dokumentu" może dotyczyć zarówno całego pliku latexowego jak i samej treści

Reply:
jest "treści dokumentu", ja to rozumiem jako content + i tak template pokazuje